### PR TITLE
fix: location

### LINKED ISSUE #204: [bug] invalid start offset encapsed
Input:
```php
call(`
command;
command;
command;
`);
```

Has `13` offset, but should `12`.

For `string` it is `12`.

```php
call('
command;
command;
command;
');
```

### PR DESCRIPTION
**Pull request by evilebottnawi**
Not all, but better.

fixes #204

### CODE REVIEW

**Comment by evilebottnawi** [comment]
Hm, it is require to fix all problem with `codeclimate` right now?

File: src/parser/class.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ module.exports = {
 
       // check T_USE trait
       if (this.token === this.tok.T_USE) {
-        result = result.concat(this.next().read_trait_use_statement());
+        result = result.concat(this.read_trait_use_statement());
```
Comment (ichiriac) [inline src/parser/class.js]: perfect

File: src/parser/class.js [hunk 2]
```diff
@@ -365,6 +365,7 @@ module.exports = {
   read_trait_use_statement: function() {
     // defines use statements
     const node = this.node("traituse");
+    this.expect(this.tok.T_USE) && this.next();
```
Comment (ichiriac) [inline src/parser/class.js]: top

**Comment by ichiriac** [comment]
no need to fix the code climate, the most important part is the code coverage.

**Pull request merged**